CreatePerson/index.ts

```diff
@@ -0,0 +1,55 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions";
+import GroupManager from "../Group/GroupManager";
+import {
+  MyPerson,
+  Person,
+  personRequirements,
+} from "../Person/PersonInterface";
+import CustomError from "../Util/customError";
+import errorHandler from "../Util/errorHandling";
+import mongooseConnection from "../Util/mongooseConnection";
+
+const httpTrigger: AzureFunction = async function (
+  context: Context,
+  req: HttpRequest
+): Promise<void> {
+  try {
+    const person: Person = {
+      name: req.body?.name,
+      favoriteColor: req.body?.favoriteColor,
+      favoriteAnimal: req.body?.favoriteAnimal,
+      favoriteFood: req.body?.favoriteFood,
+      role: req.body?.role,
+      group: req.body?.group,
+    };
```

Comment:
You already made sure there are no extraneous fields when validating with Joi. Just use req.body or its keys directly